Resources/Public/JavaScript/import.js

```diff
@@ -44,6 +44,7 @@ IgLdapSsoAuthImport = {
         IgLdapSsoAuthImport.fields.form.submit(function (e) {
             e.preventDefault(); // this will prevent from submitting the form
             var dn = $('#tx-igldapssoauth-dn').val();
+            dn=dn.replace('\\','\\\\');
```

Comment:
the escape should be done centrally in method `IgLdapSsoAuthImport.ldapImport`, and not in JS.

Reply:
Actually this fix is correct but it does not help actually importing the user with backslashes, it only properly mark the corresponding row as imported after the AJAX call.

So 8d02f4c is the real fix for importing, and this change makes the imported user become "green" without needing to reload the Backend module to see it green.